app/assets/javascripts/boot.js.erb

```diff
@@ -55,6 +55,7 @@ angular.module('probedock', [
   'probedock.tables',
   'probedock.testActivityWidget',
   'probedock.testsByWeekWidget',
+  'probedock.userChoiceCtrl',
```

Comment:
Calling this `somethingCtrl` is asking for confusion with controllers. Please rename it `userChoiceSelect` or something else.